### PR TITLE
LPS-133093 Adding new test cases

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-133093

### CODE REVIEW

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please

1. Split changes by 1 commit / file
2. Remove SAMLToolBox.macro, and add it's new macros into SAML.macro

**Comment by weller0246** [comment]
ci:help:

**Comment by weller0246** [comment]
ci:test:saml

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -921,6 +922,502 @@ definition {
...
+			samlEntityID = "samlsp",
+			spURL = "http://www.able.com:8080",
+			userEmailAddress = "test@www.able.com");
+
+		SAML.addIDPConfigurationsToServiceProvider(
+			idpAttributeMapping = "screenName,emailAddress,firstName,lastName,uuid",
+			idpEntityID = "samlidp",
+			idpKeepAliveURL = "http://localhost:8080/c/portal/saml/keep_alive",
+			idpURL = "http://localhost:8080/",
+			index = "0",
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:960]: As I remember, this needs some clever naming to reflect the business logic we set in our tests.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 2]
```diff
@@ -921,6 +922,502 @@ definition {
...
+		// Verify IdP is also logged in
+
+		Navigator.openURL();
+
+		User.viewLoggedInPG();
+	}
+
+	@description = "This is a use case for LPS-105170 TC-8: Verifying that deleting a User will sync with the SP site"
+	@priority = "3"
+	test deleteUserAfterSAMLSetupAndVerify {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1335]: All test names must start with a capital letter.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
@weller0246 I finished the review, please see my comments.
cc @ferenconodiD

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 3]
```diff
@@ -2666,7 +2666,7 @@ definition {
 		if (IsElementPresent(locator1 = "ContentRow#ENTRY_CONTENT_ROW_1")) {
 			PortletEntry.selectAll();
 
-			ClickNoError(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
+			MenuItem.clickNoError(menuItem = "Deactivate");
```
[the remark this answers is not in the document]
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/macros/User.macro:2669]: I found a message from last year, that User#deactivateCP and User#deleteCP macros are used in the tests so this change can be discarded and I'll refactor with these methods  
https://liferay.slack.com/archives/CD7939WBE/p1580321226011700?thread_ts=1580299195.010400&cid=CD7939WBE

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
@weller0246 Please create a new PR when changes are applied, thanks.

**Pull request closed without merging**